### DISCUSSION TITLE
Sealights integration

### DISCUSSION THREAD

**Discussion by ghost** [badeball/cypress-cucumber-preprocessor#882] [Q&A] [answered]
Hi Team!

Is there any possibility to pass some plugin to CucumberJS via preprocessor? I would like to integrate my Cypress tests with Sealights and I think that would be the best to use sealights-cucumber-plugin according to documentation: https://sealights.atlassian.net/wiki/spaces/SUP/pages/2846228501/Running+tests+using+Cucumber+JS+framework 
however I'm not sure how I can combine sealights-cucumber-plugin with cypress-cucumber-preprocessor.

I would much appreciate any help or suggestions.

**Comment by badeball** [comment, marked as answer]
> Is there any possibility to pass some plugin to CucumberJS via preprocessor?

No, because the preprocessor is a re-implemention mimicking CucumberJS behavior. An integration with Sealight would have to target Cypress and it won't necessarily have anything to do with this preprocessor.
